templates/vue-demo-store/layouts/account.vue

```diff
@@ -112,7 +112,7 @@ provide("swNavigation-footer-navigation", footerData);
                       @click="invokeLogout()"
                     >
                       <div i-carbon-logout text-xl inline-block />
-                      <span class="ml-3">Logout</span>
+                      <span class="ml-3 text-gray-700">Logout</span>
```

Comment:
No translation needed here?

Reply:
you are totally right, I added translations to the langs' snippets 👍🏼